mateuszwu/GPP-1
# 24 add logger

This closes #24 


## Overview
Create app logs located in file: application_logs on PROJECT_NAME\APP\bin\Debug

## Testing scenarios
- install nuget packages: serilog.sinks.file and serilog
- open admin_app add some fields to database
- go to PROJECT_NAME\AdminApp\bin\debug and open file application_logs.txt and check that there are logged changes you made.


### Notes
